client/gutenberg/extensions/prev-next/block.js

```diff
@@ -59,12 +53,25 @@ registerBlockType( 'a8c/prev-next', {
 					onChange={ next => setAttributes( { next } ) }
 				/>
 			</Fragment>
-		) : attributes.prev || attributes.next ? (
-			save( { attributes, className, isEditor: true } )
-		) : (
-			<div style={ { textAlign: 'center' } }>
-				← Add prev/next links to related posts in a series. →
-			</div>
-		),
+		);
+	}
+
+	return attributes.prev || attributes.next ? (
```

Comment:
We could go for the same approach as we did with the `isSelected` above, and have a single `if(...) return` for this one too. Will probably make it more readable than the ternary.